direct/src/interval/IntervalManager.py

```diff
@@ -133,7 +133,7 @@ def __doPythonCallbacks(self):
     def __storeInterval(self, interval, index):
         while index >= len(self.ivals):
             self.ivals.append(None)
-        assert self.ivals[index] == None or self.ivals[index] == interval
+        assert self.ivals[index] is None or self.ivals[index] == interval
```

Comment:
I don't know if separate intervals can be equal. This might be another good candidate for `is`